### PR TITLE
Validate server inputs / add server unit tests

### LINKED ISSUE #6: Getting Error: Metadata file not found. Did you include "metadata.json"
I upload just one file called metadata.json (is the name important though ?)

The same one mentioned in https://github.com/ethereum/source-verify/issues/5

And I get the following error :

```
Error: Metadata file not found. Did you include "metadata.json"
```

### PR DESCRIPTION
**Pull request by cgewecke**
**PR targets #84**
 
 + Fixes the bug discovered in #6, where submitting a single file always errored with 'Metadata not found'
 + Adds validation checks for `address` and `chain` parameters in injector
 + Adds status codes to server error responses
 + Adds unit tests for `server.ts`

### CODE REVIEW

File: src/injector.ts [hunk 1]
```diff
@@ -271,6 +299,9 @@ export default class Injector {
     files: string[]
   ) : Promise<void> {
 
+    this.validateAddresses(addresses);
```
Comment (chriseth) [inline src/injector.ts]: What if the address was left out on mainnet?

Comment (cgewecke) [reply src/injector.ts]: We don't have an exception for mainnet in the current version. What made that possible in the first PoC?

Comment (cgewecke) [reply src/injector.ts]: Ah sorry - that's what the addressDB is about. Going to open a separate issue about wiring that in.

File: src/server.ts [hunk 2]
```diff
@@ -4,25 +4,37 @@ import fileUpload from 'express-fileupload';
...
+  }]
+});
 
 const repository = './repository/';
 const port = process.env.SERVER_PORT;
 
 
 app.use(express.static('ui/dist'))
+
+// TODO: 52MB is the max file size - is this right?
```
Comment (edisinovcic) [inline src/server.ts]: Open this as issue?

Comment (cgewecke) [reply src/server.ts]: Opened #99

**Pull request merged**